react/features/talk-while-muted/constants.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * The identifier of the sound to be played when we got event for talking while muted.
+ *
+ * @type {string}
+ */
+export const TALK_WHILE_MUTED_SOUND_ID = 'TALK_WHILE_MUTED_SOUND_ID';
+
+/**
+ * The file used for the talk while muted sound notification.
+ *
+ * @type {string}
+ */
+export const TALK_WHILE_MUTED_SOUND_FILE = 'talkWhileMuted.mp3';
```

Comment:
Aren't we putting these kind of constants in sounds.js?

Reply:
We have one in feature/chat and some in features/participants.